lib/generate-index.js

```diff
@@ -78,6 +76,9 @@ function generateIndex (playbook, pages, contentCatalog, env) {
         $(this).remove()
       })
 
+      // don't index the table of contents, as this is redundant
+      $('#toc', article).remove()
+
```

Comment:
This was a distraction. I'll either put this in another commit or even better: an entirely separate PR.

Reply:
> or even better: an entirely separate PR.

Yes that's a good idea

Reply:
Removed from this PR. :+1: